webpack.config.babel.js

```diff
@@ -128,7 +131,12 @@ const formatDefineHash = defineHash =>
     value => JSON.stringify(value)
   )
 
-const basePlugins = [
+const basePlugins = (bundle_name) => ([
+  new BundleAnalyzerPlugin({
+    analyzerMode: 'static',
```

Comment:
What's the `analyzerMode` for?

Reply:
https://www.npmjs.com/package/webpack-bundle-analyzer
It creates a report file